debian/postinst

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+MYCROFT=/opt/mycroft
+MYCROFT_LOG=/var/log/mycroft
+MYCROFT_BIN=/opt/mycroft/mycroft-core/bin
+
+mkdir -p $M/skills $MYCROFT_LOG /etc/mycroft
+chown -R root:root $MYCROFT
```

Comment:
How will skill installation work when run as a normal user?

Reply:
users would install skills with root access but use the skills without root access.
The parent issue is that Mycroft doesn't work as root.